config/routes.rb

```diff
@@ -1,6 +1,22 @@
 Rails.application.routes.draw do
+  devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  # root "splash_screen#index"
```

Comment:
- [ ] Kindly remove pieces of commented codes from your codebase before deploying or pushing to Github. This makes your codebase look professional